docs/book/src/plugins/go-v2-plugin.md

```diff
@@ -0,0 +1,22 @@
+# go/v2 plugin
+
+This plugin is also used to build operators similar to [go/v3](go-v3-plugin.md). Check the following [example](https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v2) to view the files created or updated by this plugin when used with the supported subcommands.
+
+## How to use it ?
+This plugin should only be used to scaffold Go projects which are not backwards compatible with the Kubebuilder layout >= 2.0.0 and < 3.0.0. 
+Use the following command to create a new project with the go/v2 plugin:
+
+```sh
+kubebuilder init --domain tutorial.kubebuilder.io --repo tutorial.kubebuilder.io/project --plugins=go/v2
+```
+All the other subcommands supported by go/v2 plugin can be executed similarly.
+
+## Subcommands supported by the plugin
+- [x] Init
```

Comment:
```suggestion
- Init ($ kubebuilder init [OPTIONS])
```

I think we do not need the x. Also, could we supplement the follow ones? WDYT?

Reply:
Ya I think that would be better. I will make the changes